docs/source/en/task_summary.mdx

```diff
@@ -154,7 +196,61 @@ Two common types of token classification are:
 >>> from transformers import pipeline
 
 >>> classifier = pipeline(task="ner")
->>> classifier("Hugging Face is a French company based in New York City.")
+>>> preds = classifier("Hugging Face is a French company based in New York City.")
+>>> preds = [
+...     {
+...         "entity": pred["entity"],
+...         "score": round(pred["score"], 4),
+...         "index": pred["index"],
+...         "word": pred["word"],
+...         "start": pred["start"],
+...         "end": pred["end"],
+...     }
+...     for pred in preds
+... ]
+>>> preds
+[{'entity': 'I-ORG',
```

Comment:
well, this is what I mean at the beginning of my review. It's better to have each entity (here a dictionary) in a single line. Let me know if you can make it :-)

Reply:
Let me know if the new changes are better!

Reply:
Thank you @stevhliu , definitely better (at least for me, hopefully for you too :-) ) ❤️ . 

I am still not a fan of
```python
[{'score': 0.9856, 'label': 'LABEL_184'},
 {'score': 0.9976, 'label': 'snow'},
 {'score': 0.9962, 'label': 'cat'}]
```
(I usually get used to a proper indentation), but I don't want to be picky. As long as the doctest pass and Sylvain is happy too, we can keep like this.

Reply:
Definitely better for me too, thank you!